src/sonic-host-service/sonic_host_server.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""Host Service to handle docker-to-host communication"""
+
+import os
+import os.path
+import glob
+import importlib
+import sys
+
+import dbus
+import dbus.service
+import dbus.mainloop.glib
+
+from gi.repository import GObject
+
+BUS_NAME = 'org.SONiC.HostService'
```

Comment:
It will be handy to move these shared definitions to a common module. The host_service.py uses it too.

Reply:
It's actually not used in this file, I'll remove it.